xml/core_registers.xml

```diff
@@ -90,8 +90,12 @@
     </register>
 
     <register name="Debug PC" short="dpc" address="0x7b1">
-        Upon entry to debug mode, \Rdpc is written with the
-        virtual address of the instruction that encountered the exception.
+        Upon entry to debug mode, \Rdpc is updated with the virtual address of
+        the next instruction to be executed.  {\tt ebreak} instructions are an
```

Comment:
This is what Priv ISA says about $mepc:

"When a trap is taken into M-mode, mepc is written with the virtual address of the instruction that encountered the exception."

But then in the NMI section it says this: "The mepc register is written with the address of the next instruction to be executed at the time the NMI was taken,"

So I think we should break out this description into the "entry into debug mode caused by halt request (acts like NMI case)" and "entry into debug mode caused by breakpoint exception" (acts like exception), and use same phrasing.